compiler/dyno/CMakeLists.txt

```diff
@@ -38,6 +38,8 @@ set(CMAKE_CXX_STANDARD_REQUIRED True)
 set(CMAKE_C_STANDARD 11)
 set(CMAKE_C_STANDARD_REQUIRED True)
 
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
Looks like this generates a .json compilation trace of sorts? Could you put a comment about why it is here?